backend/services/job.py

```
@@ -53,31 +44,99 @@ async def schedule_agent_job(payload:RunAgentRequest):
+    job_id = str(uuid.uuid4())
+    job_results[job_id] = _build_initial_job_status(job_id, payload)
+    await redisservices.set_job_status(job_id, job_results[job_id].model_dump(mode="json"))
+    job_payload = _build_job_payload(payload, job_id, installation_access_token)
+
+    try:
+        if JOB_RUNNER_MODE in {"docker", "local"}:
+            await _enqueue_local_job(job_payload)
+        elif JOB_RUNNER_MODE == "cloud_run":
+            await _schedule_cloud_run_job(job_payload)
+        else:
+            raise ValueError(f"Unsupported JOB_RUNNER_MODE: {JOB_RUNNER_MODE}")
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)